pkg/networkmanager/interfaces.js

```diff
@@ -1441,7 +1441,7 @@ PageNetworking.prototype = {
             function has_master(iface) {
                 var connections =
                     (iface.Device ? iface.Device.AvailableConnections : iface.Connections);
-                return connections.some(function (c) { return c.Masters.length > 0; });
+                return connections.some(function (c) { return c.Masters && c.Masters.length > 0; });
```

Comment:
Should we wait for `c.Masters` to become available or do we just wait for an automatic refresh once info changes?